frontend/src/features/mapSideBar/tabs/SectionStyles.tsx

```diff
@@ -3,20 +3,6 @@ import { Col } from 'react-bootstrap';
 import { MdArrowDropDown, MdArrowDropUp } from 'react-icons/md';
 import styled from 'styled-components';
 
-export const StyledFormSection = styled.div`
-  margin: 1.5rem;
-  padding: 1.5rem;
-  background-color: white;
-  text-align: left;
```

Comment:
The removal of these styles caused a lot of snapshots to be updated.

Reply:
did you check that all affected forms look ok?

Reply:
there's a lot of margin & padding removed...

Reply:
I did check some of them and saw no difference. But having more eyes would be better